purchase.adoc

```diff
@@ -261,51 +261,80 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
 }
 ```
 
-#### How to load all purchases for a user
-
-You should never retrieve all the purchases in one request, since that can potentially put too much load on the server causing the request to be rejected or time out.
-
-Instead, you should request the purchases splitted up into "pages", using the `limit` and `lastPurchaseHash` parameters.
-
-The `limit` parameter will set the page size, and the `lastPurchaseHash` parameter sets the starting point from where to retrieve purchases. Every response from the purchase service will contain a `lastPurchaseHash` field, which can be added to the new request to retrieve the next "page".
-
-In order to load all purchases, begin with an initial request without the `lastPurchaseHash` parameter being set, this will retrieve the first _n_ purchases (where _n_ is the page size defined by the `limit` parameter):
-
-```
-GET /purchases?limit=100
-```
-
-The result will contain the first _n_ number of purchases.
-```
-{
-  "purchases": [
-    ....
-  ],
-  "firstPurchaseHash": "14233908814696HbDrnUNRji5iniGikNLiQ",
-  "lastPurchaseHash": "1423390928355zj9yI1wyTvqP46AG8NEaYg"
-}
-```
-
-To retrieve the following pages, use the `lastPurchaseHash` field in the previous response to load the next page:
-```
-GET /purchases?limit=100&lastPurchaseHash=1423390928355zj9yI1wyTvqP46AG8NEaYg
-```
-
-The result will contain the next _n_ number of purchases:
-```
+#### Example response 3 (discounts)
+```json
 {
   "purchases": [
-    ....
+    {
+      "purchaseUUID": "Qq74IOcDEeaJxS84XJ93-w",
+      "amount": 4200,
+      "country": "SE",
+      "currency": "SEK",
+      "timestamp": "2017-01-30T15:46:31.808+0000",
+      "purchaseNumber": 77,
+      "userDisplayName": "Ford Prefect",
+      "userId": 1713921,
+      "organizationId": 14184411,
+      "products": [
+        {
+          "quantity": "1",
+          "productUuid": "521c47f0-e6fd-11e6-b387-39e9b871afb0",
+          "variantUuid": "4c86e7c8-e6fd-11e6-b387-39e9b871afb0",
+          "unitPrice": 2000,
+          "name": "Foo",
+          "variantName": "",
+          "discount": {
+            "quantity": 1,
+            "amount": 500
+          },
+          "discountValue": 500,
+          "autoGenerated": false,
+          "libraryProduct": true
+        },
+        {
+          "quantity": "5",
+          "productUuid": "5bab09be-e6fd-11e6-b387-39e9b871afb0",
+          "variantUuid": "54686296-e6fd-11e6-b387-39e9b871afb0",
+          "unitPrice": 1000,
+          "name": "Bar",
+          "variantName": "",
+          "discount": {
+            "quantity": 1,
+            "percentage": 10
+          },
+          "discountValue": 500,
```

Comment:
I think it is unfortunate that the discount value is the same for both rows. It would be more easily discernible and less confusing if they had different values. I propose changing the first discount to something other than 500.